server/src/com/cloud/network/router/NetworkHelperImpl.java

```diff
@@ -171,8 +172,10 @@ public boolean sendCommandsToRouter(final VirtualRouter router, final Commands c
         try {
             answers = _agentMgr.send(router.getHostId(), cmds);
         } catch (final OperationTimedoutException e) {
-            s_logger.warn("Timed Out", e);
+            s_logger.warn("Timed out", e);
             throw new AgentUnavailableException("Unable to send commands to virtual router ", router.getHostId(), e);
+        } catch (final OperationCancelledException e) {
```

Comment:
Cancelled exception is handled without any rethrow, can it cause issues somewhere up the chain